build/conda/installer/requirements.txt

```diff
@@ -0,0 +1,9 @@
+openbb[all]
+openbb-devtools
+openbb-platform-pro-backend
+openbb-cli
+notebook
+nbclassic
+voila
```

Comment:
I suggest we don't ship voila unless there is some reasoning I'm missing.

In addition, jedi and pyright seem to both do the same job (language server) so maybe we pick one that we want to bundle and not both.